service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -1,24 +1,92 @@
-    private final CaseManagementOrderDocumentLinkNotificationHandler documentLinkNotificationHandler;
+    private final ObjectMapper mapper;
+    private final InboxLookupService inboxLookupService;
+    private final NotificationService notificationService;
+    private final RepresentativeService representativeService;
+    private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+    private final CafcassLookupConfiguration cafcassLookupConfiguration;
+    private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
 
     @EventListener
     public void sendEmailsForIssuedCaseManagementOrder(final CaseManagementOrderIssuedEvent event) {
         EventData eventData = new EventData(event);
 
-        caseLinkNotificationHandler.sendNotifications(eventData);
-        documentLinkNotificationHandler.sendNotifications(eventData,
-            event.getDocumentContents());
+        sendToLocalAuthority(eventData);
+        sendToCafcass(eventData);
+        sendToRepresentatives(eventData, DIGITAL_SERVICE);
+        sendToRepresentatives(eventData, EMAIL);
+
+        // TODO
+        // Refactor issuedOrderAdminNotificationHandler to use Java object for template so we can avoid passing byte
+        // to method
+        issuedOrderAdminNotificationHandler.sendToAdmin(eventData, event.getDocumentContents(), CMO);
+    }
+
+    private void sendToLocalAuthority(final EventData eventData) {
+        final IssuedCMOTemplate localAuthorityNotificationParameters = caseManagementOrderEmailContentProvider
+            .buildCMOIssuedNotificationParameters(eventData.getCaseDetails(), DIGITAL_SERVICE);
+
+        final String email = inboxLookupService.getNotificationRecipientEmail(eventData.getCaseDetails(),
+            eventData.getLocalAuthorityCode());
+
+        notificationService.sendEmail(CMO_ORDER_ISSUED_NOTIFICATION_TEMPLATE, email,
+            localAuthorityNotificationParameters, eventData.getReference());
+    }
+
+    private void sendToCafcass(final EventData eventData) {
+        final IssuedCMOTemplate cafcassParameters =
+            caseManagementOrderEmailContentProvider.buildCMOIssuedNotificationParameters(
+                eventData.getCaseDetails(), EMAIL);
+
+        final String cafcassEmail = cafcassLookupConfiguration.getCafcass(eventData.getLocalAuthorityCode()).getEmail();
+
+        notificationService.sendEmail(CMO_ORDER_ISSUED_NOTIFICATION_TEMPLATE, cafcassEmail, cafcassParameters,
+            eventData.getReference());
+    }
+
+    private void sendToRepresentatives(final EventData eventData, RepresentativeServingPreferences servingPreference) {
+        CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+        List<Representative> representatives = representativeService.getRepresentativesByServedPreference(
+            caseData.getRepresentatives(), servingPreference);
+
+        representatives.stream()
```

Comment:
If there is an issue with sending one of the emails this would block all emails. Is that what we want? Do we want it to be done in parallel? Would that be out of the scope of the ticket?

Reply:
Agreed to make tech debt.